pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -888,9 +888,16 @@ func getTaskrunWorkspaces(pr *v1beta1.PipelineRun, rprt *resources.ResolvedPipel
 	}
 	for _, ws := range rprt.PipelineTask.Workspaces {
 		taskWorkspaceName, pipelineTaskSubPath, pipelineWorkspaceName := ws.Name, ws.SubPath, ws.Workspace
-		if b, hasBinding := pipelineRunWorkspaces[pipelineWorkspaceName]; hasBinding {
+
+		pipelineWorkspace := pipelineWorkspaceName
```

Comment:
not sure there is a need to create a new variable for this?

Reply:
think so, otherwise I'd have to include more if's switching between pipelineWorkspaceName and taskWorkspaceName variables